network/cert_verifier.go

```diff
@@ -28,16 +32,139 @@ var (
 	ErrEmptyCertChain = errors.New("empty verified certificates chain")
 )
 
+var (
+	tlsOcspURL                      string
+	tlsOcspClientURL                string
+	tlsOcspDbURL                    string
+	tlsOcspRequired                 string
+	tlsOcspFromCert                 string
+	tlsOcspCheckOnlyLeafCertificate bool
+	tlsCrlURL                       string
+	tlsCrlClientURL                 string
+	tlsCrlDbURL                     string
+	tlsCrlFromCert                  string
+	tlsCrlCheckOnlyLeafCertificate  bool
+	tlsCrlCacheSize                 uint
+	tlsCrlCacheTime                 uint
+)
+
+// RegisterCertVerifierArgs register CLI args tls_ocsp_url|tls_ocsp_client_url|tls_ocsp_database_url|tls_ocsp_required|tls_ocsp_from_cert|tls_ocsp_check_only_leaf_certificate|tls_crl_url|tls_crl_client_url|tls_crl_database_url|tls_crl_from_cert|tls_crl_check_only_leaf_certificate|tls_crl_cache_size|tls_crl_cache_time which allow to get CertVerifier by NewCertVerifierFromArgs|NewClientCertVerifierFromArgs|NewDatabaseCertVerifierFromArgs functions
+func RegisterCertVerifierArgs(separate_client_db_urls bool) {
```

Comment:
I have a point to make and I'm going to hit it every time.

It's hard to understand why AcraConnector needs
```go
network.RegisterCertVerifierArgs(false)
```
and AcraServer uses
```go
network.RegisterCertVerifierArgs(true)
```
Yes, I lead a crusade over boolean flags.

For example,
```go
func RegisterCertVerifierArgs() {
        registerCertVerifierArgs(false)
}

func RegisterCertVerifierArgsWithSeparateClientAndDatabase() {
        registerCertVerifierArgs(true)
}

func registerCertVerifierArgs(separate_client_db_urls bool) {
        // whatever we have now
}
```
makes for more readable code at the use site, despite looking like it came from the Java land.

Reply:
OK, I don't think two more tiny functions will make it too complex